src/util/util_bit.h

```diff
@@ -320,6 +320,21 @@ namespace dxvk::bit {
       return get(idx);
     }
 
+    constexpr void setN(uint32_t bits) {
+      uint32_t dwords = align(bits, 32) / 32;
```

Comment:
You can get rid of this `align` and just use `dwords` for the loop in both paths.

As in the first path, it will always be aligned to 32, so nothing changes, and in the second, the un-aligned value of dwords will always be the value of aligned dwords - 1

Reply:
Maybe call it `fullDwords` instead or something